arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2667,6 +2667,10 @@
           "type": "boolean",
           "description": "Gets or sets whether the extension handler should be automatically upgraded across minor versions."
         },
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "Gets or sets whether the extension handler should be forced to re-run even if the extension configuration has not changed."
```

Comment:
Get means it gives the value of the parameter during response body.  I just followed other descriptions.